README.md

```diff
@@ -80,32 +80,61 @@ metric exporters, service monitors, and custom dashboards.
 See the [logging README](logging/README.md) to deploy the logging components,
 including Fluent Bit, Elasticsearch, and Kibana.
 
-## Miscellaneous
+## Customization
 
-### Customization
+The process of customizing the monitoring and logging deployments consists of: 
+- creating the location for your local customization files
+- specifying customization variables and parameters in the customization files
+- using the `USER_DIR` environment variable to specify the location of the customization files
 
-The components that are deployed are highly customizable. The recommended
-approach for customization is to set a `USER_DIR` environment variable. `USER_DIR`
-points to a directory containing user-modified customization files which can exist
-outside of your repository. You can use customization files to check in
-environment-specific customizations into a separate Git repository, for example.
+### Create the Location for Customization Files
 
-`USER_DIR` must refer to a directory. The directory can include any or all of
-the following files in the structure below. Additional content is allowed, but
-will be ignored.
+The `USER_DIR` environment variable enables you to use a directory outside of the local repository to contain the customization files. By using a location outside of the repository, you can preserve your customizations when you deploy new versions of the monitoring and logging applications. You could also put the customization files in a location that is under version control.
+
+For example, to create a customization directory named *my_custom_files* that contains directories for monitoring and logging customization files:
+
+```bash
+mkdir -p ~/my_custom_files/monitoring
```

Comment:
Suggestion - replace 'my_custom_files' with 'my-v4m-user-dir' or something more descriptive of what it is.

Reply:
Done